edi-adapter-model/src/main/kotlin/no/nav/emottak/ediadapter/model/serialization/Serializers.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.emottak.ediadapter.model.serialization
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import java.time.Instant
+import kotlin.uuid.ExperimentalUuidApi
+import kotlin.uuid.Uuid
+
+@OptIn(ExperimentalUuidApi::class)
+object UuidSerializer : KSerializer<Uuid> {
```

Comment:
Det skal ikke være nødvendig og ha en custom serializer for Uuid'er. Det finnes allerede i _kotlinx-serialization_. Ref: https://github.com/Kotlin/kotlinx.serialization/blob/master/core/commonMain/src/kotlinx/serialization/builtins/BuiltinSerializers.kt#L286